fury/actor.py

```diff
@@ -1221,6 +1222,91 @@ def display(self, x=None, y=None, z=None):
     return peak_actor
 
 
+def peak(peaks_dirs, peaks_values=None, mask=None, affine=None, colors=None,
+         linewidth=1, lookup_colormap=None):
+    """Visualize peak directions as given from ``peaks_from_model``.
+
+    Parameters
+    ----------
+    peaks_dirs : ndarray
+        Peak directions. The shape of the array should be (X, Y, Z, D, 3).
+    peaks_values : ndarray
+        Peak values. The shape of the array should be (X, Y, Z, D)
+    affine : array
+        4x4 transformation array from native coordinates to world coordinates
+    mask : ndarray
+        3D mask
+    colors : tuple or None
+        Default None. If None then every peak gets an orientation color
+        in similarity to a DEC map.
+    lookup_colormap : vtkLookupTable, optional
+        Add a default lookup table to the colormap. Default is None which calls
+        :func:`fury.actor.colormap_lookup_table`.
+    linewidth : float, optional
+        Line thickness. Default is 1.
+
+    Returns
+    -------
+    actor : PeakActor
+        vtkActor or vtkLODActor representing the peaks directions and/or
+        magnitudes.
+
+    Examples
+    ----------
+    >>> from fury import actor, window
+    >>> import numpy as np
+    >>> scene = window.Scene()
+    >>> peak_dirs = np.random.rand(3, 3, 3, 3, 3)
+    >>> c = actor.peak(peak_dirs)
+    >>> scene.add(c)
+    >>> #window.show(scene)
+
+    """
+    peaks_dirs = np.asarray(peaks_dirs)
```

Comment:
why you need `np.asarray`?  in the documentation, `peaks_dirs` is already a ndarray

Reply:
Changed